packages/valory/contracts/mech_marketplace/build/mech.json

```diff
@@ -7,37 +7,80 @@
       "inputs": [
         {
           "internalType": "address",
-          "name": "_stakingFactory",
```

Comment:
Why are we changing this abi? Shouldnt we be adding a new ABI in a new file for MM?

Reply:
https://github.com/valory-xyz/IEKit/pull/274#discussion_r2028318733